projects/fusion-ui/components/file-drag-and-drop/file-drag-and-drop.directive.ts

```diff
@@ -95,7 +103,7 @@ export class FileDragAndDropDirective implements OnInit, AfterViewInit, OnDestro
     onDragEnter(event) {
         event.preventDefault();
         event.stopPropagation();
-        if (this._disabled) {
+        if (this._disabled || this._loading) {
```

Comment:
Repeated code please move to a private function in methods: onDragEnter, onDragLeave, and onDrag

Reply:
ok

Reply:
ok